build.sh

```diff
@@ -255,23 +255,25 @@ build_gmake() {
     cp make gmake
 }
 
-JDK_DIR=openjdk-8u72b15-bsd-port-20160220
+# We use the JDK we just built for all consequent Java compilation (and as a
+# basis for the Graal compiler).
 case `uname` in
-    Linux)
-      JDK_JAVAC=${wrkdir}/openjdk/build/linux-x86_64-normal-server-release/jdk/bin/javac;;
-    OpenBSD)
-      JDK_JAVAC=${wrkdir}/openjdk/build/bsd-x86_64-normal-server-release/jdk/bin/javac;;
+    Linux)   OUR_JAVA_HOME=${wrkdir}/openjdk/build/linux-x86_64-normal-server-release/images/j2sdk-image/;;
```

Comment:
Can we find a nicer name than `OUR_`? I always find it really ambiguous. Maybe just call it `JAVAC`? [`JDK_` seems a bit redundant.]